jolt-core/src/main/java/io/joltcommunity/jolt/Sortr.java

```diff
@@ -80,8 +80,8 @@ private static class JsonKeyComparator implements Comparator<String> {
         @Override
         public int compare(String a, String b) {
 
-            boolean aTilde = (a.length() > 0 && a.charAt(0) == '~');
-            boolean bTilde = (b.length() > 0 && b.charAt(0) == '~');
+            boolean aTilde = (!a.isEmpty() && a.charAt(0) == '~');
+            boolean bTilde = (!b.isEmpty() && b.charAt(0) == '~');
```

Comment:
Due to json's keys can't be duplicated, so we CAN'T cover a and b are both empty case.